src/toggle/styles.scss

```diff
@@ -23,6 +23,9 @@ $shadow-color: rgba(0, 0, 0, 0.25);
 }
 
 .toggle-control {
+  // this is a "drawing" element, and therefore we want it (and children) to retain
+  // background colors in high-contrast/forced-color mode
+  forced-color-adjust: none;
```

Comment:
Safari doesn't have support for this property yet.

Reply:
as far as I can tell that means that forced-color/high-contrast theme isn't a thing in Safari at all, so it doesn't matter that it's not supported as there's no way of putting the browser into a mode where the property would need to be supported

Reply:
Safari supports the forced-color media query but I couldn't quite figure out how to emulate this correctly with Safari either. Closest I got was increased contrast mode via MacOS but that didn't really do the same thing.